pkg/util/kubeconfig.go

```diff
@@ -325,3 +325,16 @@ func UnsetCurrentContext(filename, machineName string) error {
 	}
 	return nil
 }
+
+//Fetch all the profiles and return
+func GetProfiles(filename string) ([]string, error) {
+	config, err := ReadConfigOrNew(filename)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error getting kubeconfig status")
+	}
+	profiles := make([]string, 0, 4)
+	for context, _ := range config.Contexts {
+		profiles = append(profiles, context)
```

Comment:
This is getting all kubecontexts... Inclduing the ones not created by minikube.

Reply:
> @medyagh the contexts which I get via the above way and the profiles present in ~/.minikube/profiles directory. Although it contains the profile created by kubernetes (`minikube`), shouldn't it be shown too? Because it is also there under the profiles directory.

that code will get all the kubectl contexts, so for example if I have a GKE cluster or another k8s cluster, there will be a list of contexts, and when you create minikube profile it will create another context.

if you get list of all contexts and show it as list of profiles, it will show my gke cluster or my other k8s cluster as a minikube profile , so it is wrong